except_star.md

```diff
@@ -483,6 +483,30 @@ errors is error prone.
 
 We can consider allowing some of them in future versions of Python.
 
+## The Traceback of an Exception Group
+
+For regular exceptions, the traceback represents a simple path of frames,
+from the frame in which the exception was raised to the frame in which it was
+was caught or, if it hasn't been caught yet, the frame that the program's
+execution is currently in. The list is constructed by the interpreter which
+appends a frame to the traceback of the 'current exception' (the one returned
+by `sys.exc_info()`) if one exists. To support efficient appends, the links
+in a traceback's list of frames are from the oldest to the newest frame.
+Appending a new frame is then simply a matter of inserting a new head to the
+linked list referenced from the exception's `__traceback__` field. Crucially,
+the traceback is immutable in the sense that once frames are added they are
+no longer modified.
```

Comment:
It's really the singly-linked list that's no longer modified, right? So the list is immutable.